src/utils/common-functions.js

```diff
@@ -125,7 +125,7 @@ export const parseStateTimeseries = ({states_daily: data}) => {
   for (let i = 0; i < data.length; i += 3) {
     const date = new Date(data[i].date);
     // Skip data from the current day
-    if (new Date(date.toDateString()) < new Date(today.toDateString())) {
+    if (date < today) {
```

Comment:
Hey, I deliberately wrote this ugly piece of code to check only for the date and not the time. Otherwise, it was still displaying today's data

Reply:
I'm sorry @shuklaayush 🤦‍♂️ I will replace the check back.